app/__init__.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask, render_template, redirect, url_for, request
+from .forms.DeployForm import DeployForm
```

Comment:
Yes, there's something for Atom... I've been too lazy to properly configure it 😳 I will do as well as the test.